llvm/utils/git/requirements_linting.txt.in

```diff
@@ -0,0 +1 @@
+PyGithub==1.59.1
```

Comment:
Is there a reason this (and the lockfile) cannot be shared with the existing job?

Reply:
Ah, right. Forgot about that. Keeping it separate I guess makes sense then.